README.md

```diff
@@ -1230,6 +1229,11 @@ Where to discover new Python libraries.
 * [@pypi](https://twitter.com/pypi)
 * [@pythontrending](https://twitter.com/pythontrending)
 * [@PythonWeekly](https://twitter.com/PythonWeekly)
+ 
+## Podcasts
+
+* [Podcast.\__init__](http://podcastinit.com/) - Podcast
```

Comment:
Its name is **Podcast.__init__** and the ` - Podcast` suffix could be removed.

Reply:
Thank you guys! Willie Parish